README.md

```diff
@@ -1105,7 +1105,7 @@ See [CONTRIBUTING.md](https://github.com/carrierwaveuploader/carrierwave/blob/ma
 
 The MIT License (MIT)
 
-Copyright (c) 2008-2015 Jonas Nicklas
+Copyright (c) 2008-2023 Jonas Nicklas
```

Comment:
```suggestion
Copyright (c) 2008 Jonas Nicklas
```

Isn't just using the first publication date enough? It's pointless to need to update the number every year...

> DO YOU CHANGE THE YEARS STATED ON YOUR BLOG AND WEBSITE?
> 
> Use the earliest date of publication. For example, if you first published on your blog in 2012, when you amend the blog content or add to it, continue to use the 2012 date or use a range of dates such as 2012–2022.

https://www.copyrightlaws.com/copyright-symbol-notice-year/

Reply:
Indeed it is. I updated this PR. Thank you.